src/include/optimizer/cost.h

```diff
@@ -67,6 +67,8 @@ extern PGDLLIMPORT bool enable_parallel_append;
 extern PGDLLIMPORT bool enable_parallel_hash;
 extern PGDLLIMPORT bool enable_partition_pruning;
 extern PGDLLIMPORT bool enable_async_append;
+extern PGDLLIMPORT bool enable_seqscan_prefetch;
+extern PGDLLIMPORT int seqscan_prefetch_buffers;
```

Comment:
It was laced here because this parameter is similar with enable_seqscan,
Yes, internally this parameters belongs to different groups:  enable_seqscan is parameter to optimizer, while enable_seqsan_prefetch can be considered as parameter to executor. But fro users pont of vie them looks similar.